src/overlays/actors/ovl_En_Nb/z_en_nb.c

```diff
@@ -21,255 +169,1401 @@ const ActorInit En_Nb_InitVars = {
+    EnNb_FrameUpdateMatrix(this);
+    EnNb_UpdateEyes(this);
+    EnNb_SetupLightOrb(this, globalCtx);
+    func_80AB1210(this, globalCtx);
+}
+
+void EnNb_DrawTransparency(EnNb* this, GlobalContext* globalCtx) {
+    s32 pad[2];
+    s16 eyeSegIdx = this->eyeIdx;
+    UNK_PTR addr = sEyeSegments[eyeSegIdx];
+    SkelAnime* skelAnime = &this->skelAnime;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_nb_inKenjyanomaDemo02.c", 263);
+    func_80093D84(globalCtx->state.gfxCtx);
+
+    gSPSegment(oGfxCtx->polyXlu.p++, 0x08, SEGMENTED_TO_VIRTUAL(addr));
+    gSPSegment(oGfxCtx->polyXlu.p++, 0x09, SEGMENTED_TO_VIRTUAL(addr));
+    gDPSetEnvColor(oGfxCtx->polyXlu.p++, 0, 0, 0, this->alpha);
+    gSPSegment(oGfxCtx->polyXlu.p++, 0x0C, &D_80116280[0]);
+
+    oGfxCtx->polyXlu.p = SkelAnime_DrawSV2(globalCtx, skelAnime->skeleton, skelAnime->limbDrawTbl,
+                                           skelAnime->dListCount, NULL, NULL, NULL, oGfxCtx->polyXlu.p);
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_nb_inKenjyanomaDemo02.c", 290);
+}
+
+void func_80AB1284(EnNb* this, GlobalContext* globalCtx);
+
+void EnNb_InitKidnap(EnNb* this, GlobalContext* globalCtx) {
+    EnNb_SetCurrentAnim(this, &D_06001E7C, 0, 0.0f, 0);
+    this->action = NB_KIDNAPPED;
+    this->actor.shape.unk_14 = 0;
+    gSaveContext.eventChkInf[9] |= 0x20;
+}
+
+void EnNb_PlayCrySFX(EnNb* this, GlobalContext* globalCtx) {
+    if (globalCtx->csCtx.frames == 3) {
+        func_80078914(&this->actor.projectedPos, NA_SE_VO_NB_CRY_0);
+    }
+}
+
+void EnNb_PlayAgonySFX(EnNb* this, GlobalContext* globalCtx) {
+    if (globalCtx->csCtx.frames == 420) {
+        func_80078914(&this->actor.projectedPos, NA_SE_VO_NB_AGONY);
+    }
+}
+
+void EnNb_SetPosInPortal(EnNb* this, GlobalContext* globalCtx) {
+    CsCmdActorAction* csCmdNPCAction = EnNb_GetNpcCsAction(globalCtx, 1);
+    Vec3f* pos = &this->actor.posRot.pos;
+    f32 f0;
+    s32 pad;
+    Vec3f startPos;
+    Vec3f endPos;
+    if (csCmdNPCAction != 0) {
+        f0 = func_8006F9BC(csCmdNPCAction->endFrame, csCmdNPCAction->startFrame, globalCtx->csCtx.frames, 4, 4);
+        startPos.x = csCmdNPCAction->startPos.x;
+        startPos.y = csCmdNPCAction->startPos.y;
+        startPos.z = csCmdNPCAction->startPos.z;
+        endPos.x = csCmdNPCAction->endPos.x;
+        endPos.y = csCmdNPCAction->endPos.y;
+        endPos.z = csCmdNPCAction->endPos.z;
+        pos->x = ((endPos.x - startPos.x) * f0) + startPos.x;
+        pos->y = ((endPos.y - startPos.y) * f0) + startPos.y;
+        pos->z = ((endPos.z - startPos.z) * f0) + startPos.z;
+    }
+}
+
+void EnNb_SetupCaptureCutsceneState(EnNb* this, GlobalContext* globalCtx) {
+    EnNb_SetupCsPosRot(this, globalCtx, 1);
+    this->action = NB_KIDNAPPED;
+    this->drawMode = NB_DRAW_NOTHING;
+    this->actor.shape.unk_14 = 0;
+}
+
+void EnNb_SetRaisedArmCaptureAnim(EnNb* this, s32 animFinished) {
+    AnimationHeader* animation = &D_06001350;
+    if (animFinished) {
```

Comment:
```suggestion
    AnimationHeader* animation = &D_06001350;
    
    if (animFinished) {
```
empty line after declarations, in other functions too